docs/book/toc.md

```diff
@@ -29,117 +28,120 @@
-  * [Use the Secret Store](platform-guide/set-up-your-mlops-platform/use-the-secret-store/use-the-secret-store.md)
-    * [Interact with secrets](platform-guide/set-up-your-mlops-platform/use-the-secret-store/interact-with-secrets.md)
-    * [Custom secret stores](platform-guide/set-up-your-mlops-platform/use-the-secret-store/custom-secret-stores.md)
-  * [Connect ZenML to infrastructure](platform-guide/set-up-your-mlops-platform/connect-zenml-to-infrastructure.md)
-    * [Service Connectors guide](platform-guide/set-up-your-mlops-platform/connect-zenml-to-infrastructure/service-connectors-guide.md)
-    * [Security best practices](platform-guide/set-up-your-mlops-platform/connect-zenml-to-infrastructure/best-security-practices.md)
-    * [Docker Service Connector](platform-guide/set-up-your-mlops-platform/connect-zenml-to-infrastructure/docker-service-connector.md)
-    * [Kubernetes Service Connector](platform-guide/set-up-your-mlops-platform/connect-zenml-to-infrastructure/kubernetes-service-connector.md)
-    * [AWS Service Connector](platform-guide/set-up-your-mlops-platform/connect-zenml-to-infrastructure/aws-service-connector.md)
-    * [GCP Service Connector](platform-guide/set-up-your-mlops-platform/connect-zenml-to-infrastructure/gcp-service-connector.md)
-    * [Azure Service Connector](platform-guide/set-up-your-mlops-platform/connect-zenml-to-infrastructure/azure-service-connector.md)
-  * [Deploy & set up a cloud stack](platform-guide/set-up-your-mlops-platform/deploy-and-set-up-a-cloud-stack/deploy-and-set-up-a-cloud-stack.md)
-    * [Deploy a stack component](platform-guide/set-up-your-mlops-platform/deploy-and-set-up-a-cloud-stack/deploy-a-stack-component.md)
-    * [Deploy a stack using Stack Recipes](platform-guide/set-up-your-mlops-platform/deploy-and-set-up-a-cloud-stack/deploy-a-stack-using-stack-recipes.md)
-    * [Contribute flavors or components](platform-guide/set-up-your-mlops-platform/deploy-and-set-up-a-cloud-stack/contribute-flavors-or-components.md)
-  * [Productionalize with CI/CD/CT](platform-guide/set-up-your-mlops-platform/productionalize-with-ci-cd-ct.md)
-  * [Implement a custom stack component](platform-guide/set-up-your-mlops-platform/implement-a-custom-stack-component.md)
-  * [Implement a custom integration](platform-guide/set-up-your-mlops-platform/implement-a-custom-integration.md)
-* [🔧 Manage the deployed services](platform-guide/manage-the-deployed-services/manage-the-deployed-services.md)
-  * [Upgrade the version of the ZenML server](platform-guide/manage-the-deployed-services/upgrade-the-version-of-the-zenml-server.md)
-  * [Troubleshoot the deployed server](platform-guide/manage-the-deployed-services/troubleshoot-your-deployed-server.md)
-  * [Troubleshoot stack components](platform-guide/manage-the-deployed-services/troubleshoot-stack-components.md)
+* [📜 Component guide](stacks-and-components/component-guide/component-guide.md)
```

Comment:
No ideas for now, but have been thinking for a while that this deserves a bit more of an enticing name. 
- component encyclopedia
- component reference guide
- component a-z
- component 101
or something else to make it clear that there's some stuff going on in there!

Reply:
If it is alright, for the time being, I would keep it as it is.